### ISSUE TITLE
Recovered from panic in HTTP handler in ping@internal

### ISSUE THREAD

**Issue by scratchmex** [containous/traefik#6297]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?
http get request to `mydomain.dev/ping`

### What did you expect to see?
OK status from ping

### What did you see instead?
Internal Server Error

### Output of `traefik version`: (_What version of Traefik are you using?_)

```console
$ docker run traefik version
Version:      2.1.3
Codename:     cantal
Go version:   go1.13.6
Built:        2020-01-21T17:30:29Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

**traefik.yml:**
```yml
api:
  dashboard: true

log:
  level: DEBUG

entryPoints:
  http:
    address: ":80"
  https:
    address: ":443"

providers:
  docker:
    endpoint: "unix:///var/run/docker.sock"
    exposedByDefault: false

certificatesResolvers:
  leresolver:
    acme:
      email: me@mydomain.dev
      storage: acme.json
      httpChallenge:
        # used during the challenge
        entryPoint: http
```

**docker-compose.yml:**
```yml
version: '3.7'

services:
  traefik:
    image: traefik:latest
    container_name: traefik
    restart: unless-stopped
    networks:
      - traefik
    ports:
      - 80:80
      - 443:443
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /var/run/docker.sock:/var/run/docker.sock:ro
      - ./traefik.yml:/traefik.yml:ro
      - ./acme.json:/acme.json
    labels:
      - "traefik.enable=true"
      # ping service
      - "traefik.http.routers.ping.rule=Host(`mydomain.dev`)&&Path(`/ping`)"
      - "traefik.http.routers.ping.service=ping@internal"
      - "traefik.http.routers.ping.tls.certresolver=leresolver"
      - "traefik.http.routers.ping.tls=true"
      # middleware redirect
      - "traefik.http.middlewares.redirect-to-https.redirectscheme.scheme=https"
      # global redirect to https
      - "traefik.http.routers.redirs.rule=hostregexp(`{host:.+}`)"
      - "traefik.http.routers.redirs.entrypoints=http"
      - "traefik.http.routers.redirs.middlewares=redirect-to-https"
      # traefik dashboard
      - "traefik.http.routers.traefik.entrypoints=https"
      - "traefik.http.routers.traefik.rule=Host(`mydomain.dev`)&&PathPrefix(`/traefik`)"
      - "traefik.http.routers.traefik.tls=true"
      - "traefik.http.routers.traefik.tls.certresolver=leresolver"
      - "traefik.http.routers.traefik.service=api@internal"
      # setup middlewares
      - "traefik.http.middlewares.simpleauth.basicauth.users=admin:<hashhh>"
      - "traefik.http.middlewares.traefik_fixprefix.stripprefix.prefixes=/traefik"
      # register middlewares
      - "traefik.http.routers.traefik.middlewares=traefik_fixprefix,simpleauth"

networks:
  traefik:
    name: traefik
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

<details>
<summary>logs</summary>

```console
$ docker-compose up
Starting traefik ... done
Attaching to traefik
traefik    | time="2020-02-09T17:43:15Z" level=info msg="Configuration loaded from file: /traefik.yml"
traefik    | time="2020-02-09T17:43:15Z" level=info msg="Traefik version 2.1.3 built on 2020-01-21T17:30:29Z"
traefik    | time="2020-02-09T17:43:15Z" level=debug msg="Static configuration loaded {\"global\":{\"checkNewVersion\":true},\"serversTransport\":{\"maxIdleConnsPerHost\":200},\"entryPoints\":{\"http\":{\"address\":\":80\",\"transport\":{\"lifeCycle\":{\"graceTimeOut\":10000000000},\"respondingTimeouts\":{\"idleTimeout\":180000000000}},\"forwardedHeaders\":{}},\"https\":{\"address\":\":443\",\"transport\":{\"lifeCycle\":{\"graceTimeOut\":10000000000},\"respondingTimeouts\":{\"idleTimeout\":180000000000}},\"forwardedHeaders\":{}}},\"providers\":{\"providersThrottleDuration\":2000000000,\"docker\":{\"watch\":true,\"endpoint\":\"unix:///var/run/docker.sock\",\"defaultRule\":\"Host(`{{ normalize .Name }}`)\",\"swarmModeRefreshSeconds\":15000000000}},\"api\":{\"dashboard\":true},\"log\":{\"level\":\"DEBUG\",\"format\":\"common\"},\"certificatesResolvers\":{\"leresolver\":{\"acme\":{\"email\":\"me@ivgon.me\",\"caServer\":\"https://acme-v02.api.letsencrypt.org/directory\",\"storage\":\"acme.json\",\"keyType\":\"RSA4096\",\"httpChallenge\":{\"entryPoint\":\"http\"}}}}}"  traefik    | time="2020-02-09T17:43:15Z" level=info msg="\nStats collection is disabled.\nHelp us improve Traefik by turning this feature on :)\nMore details on: https://docs.traefik.io/v2.0/contributing/data-collection/\n"
traefik    | time="2020-02-09T17:43:15Z" level=info msg="Starting provider aggregator.ProviderAggregator {}"
traefik    | time="2020-02-09T17:43:15Z" level=debug msg="Start TCP Server" entryPointName=http
traefik    | time="2020-02-09T17:43:15Z" level=debug msg="Start TCP Server" entryPointName=https
traefik    | time="2020-02-09T17:43:15Z" level=info msg="Starting provider *acme.Provider {\"email\":\"me@ivgon.me\",\"caServer\":\"https://acme-v02.api.letsencrypt.org/directory\",\"storage\":\"acme.json\",\"keyType\":\"RSA4096\",\"httpChallenge\":{\"entryPoint\":\"http\"},\"ResolverName\":\"leresolver\",\"store\":{},\"ChallengeStore\":{}}"
traefik    | time="2020-02-09T17:43:15Z" level=info msg="Testing certificate renew..." providerName=leresolver.acme
traefik    | time="2020-02-09T17:43:15Z" level=info msg="Starting provider *docker.Provider {\"watch\":true,\"endpoint\":\"unix:///var/run/docker.sock\",\"defaultRule\":\"Host(`{{ normalize .Name }}`)\",\"swarmModeRefreshSeconds\":15000000000}"
traefik    | time="2020-02-09T17:43:15Z" level=info msg="Starting provider *traefik.Provider {}"
traefik    | time="2020-02-09T17:43:15Z" level=debug msg="Configuration received from provider leresolver.acme: {\"http\":{},\"tls\":{}}" providerName=leresolver.acme
traefik    | time="2020-02-09T17:43:15Z" level=debug msg="Configuration received from provider internal: {\"http\":{\"services\":{\"api\":{},\"dashboard\":{}}},\"tcp\":{},\"tls\":{}}" providerName=internal
traefik    | time="2020-02-09T17:43:15Z" level=debug msg="Adding certificate for domain(s) who.mydomain.dev,mydomain.dev"
traefik    | time="2020-02-09T17:43:15Z" level=debug msg="Adding certificate for domain(s) dash.mydomain.dev"
traefik    | time="2020-02-09T17:43:15Z" level=debug msg="Adding certificate for domain(s) traefik.mydomain.dev"
traefik    | time="2020-02-09T17:43:15Z" level=debug msg="Adding certificate for domain(s) portainer.mydomain.dev,mydomain.dev"
traefik    | time="2020-02-09T17:43:15Z" level=debug msg="Adding certificate for domain(s) blog.mydomain.dev,mydomain.dev"
traefik    | time="2020-02-09T17:43:15Z" level=debug msg="No default certificate, generating one"
traefik    | time="2020-02-09T17:43:15Z" level=debug msg="Provider connection established with docker 19.03.5 (API 1.40)" providerName=docker
traefik    | time="2020-02-09T17:43:15Z" level=error msg="field not found, node: certresolver" providerName=docker container=ghost-blog-6708801a6f9def1fb906995a3ee6bc5fa8960696701f129965ed7651403f4870
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Configuration received from provider docker: {\"http\":{\"routers\":{\"ping\":{\"service\":\"ping@internal\",\"rule\":\"Host(`mydomain.dev`)\\u0026\\u0026Path(`/ping`)\",\"tls\":{\"certResolver\":\"leresolver\"}},\"redirs\":{\"entryPoints\":[\"http\"],\"middlewares\":[\"redirect-to-https\"],\"service\":\"traefik-traefik\",\"rule\":\"hostregexp(`{host:.+}`)\"},\"traefik\":{\"entryPoints\":[\"https\"],\"middlewares\":[\"traefik_fixprefix\",\"simpleauth\"],\"service\":\"api@internal\",\"rule\":\"Host(`mydomain.dev`)\\u0026\\u0026PathPrefix(`/traefik`)\",\"tls\":{\"certResolver\":\"leresolver\"}},\"whoami\":{\"entryPoints\":[\"https\"],\"middlewares\":[\"whoami_fixprefix\"],\"service\":\"whoami-whoami\",\"rule\":\"Host(`mydomain.dev`)\\u0026\\u0026PathPrefix(`/who`)\",\"tls\":{\"certResolver\":\"leresolver\"}}},\"middlewares\":{\"redirect-to-https\":{\"redirectScheme\":{\"scheme\":\"https\"}},\"simpleauth\":{\"basicAuth\":{\"users\":[\"admin:$2y$05$YQS7J9yA23LiJiK5tduaoONLTwUlZzTcZTN9P5QRc35LH94KNieYe\"]}},\"traefik_fixprefix\":{\"stripPrefix\":{\"prefixes\":[\"/traefik\"],\"forceSlash\":true}},\"whoami_fixprefix\":{\"stripPrefix\":{\"prefixes\":[\"/who\"],\"forceSlash\":true}}},\"services\":{\"traefik-traefik\":{\"loadBalancer\":{\"servers\":[{\"url\":\"http://172.28.0.2:80\"}],\"passHostHeader\":true}},\"whoami-whoami\":{\"loadBalancer\":{\"servers\":[{\"url\":\"http://172.28.0.5:80\"}],\"passHostHeader\":true}}}},\"tcp\":{}}" providerName=docker
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding certificate for domain(s) who.mydomain.dev,mydomain.dev"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding certificate for domain(s) dash.mydomain.dev"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding certificate for domain(s) traefik.mydomain.dev"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding certificate for domain(s) portainer.mydomain.dev,mydomain.dev"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding certificate for domain(s) blog.mydomain.dev,mydomain.dev"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="No default certificate, generating one"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding certificate for domain(s) who.mydomain.dev,mydomain.dev"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding certificate for domain(s) dash.mydomain.dev"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding certificate for domain(s) traefik.mydomain.dev"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding certificate for domain(s) portainer.mydomain.dev,mydomain.dev"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding certificate for domain(s) blog.mydomain.dev,mydomain.dev"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="No default certificate, generating one"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating middleware" middlewareType=Pipelining serviceName=traefik-traefik entryPointName=http routerName=redirs@docker middlewareName=pipelining
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating load-balancer" serviceName=traefik-traefik entryPointName=http routerName=redirs@docker
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating server 0 http://172.28.0.2:80" routerName=redirs@docker serviceName=traefik-traefik entryPointName=http serverName=0
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Added outgoing tracing middleware traefik-traefik" entryPointName=http routerName=redirs@docker middlewareName=tracing middlewareType=TracingForwarder
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating middleware" entryPointName=http middlewareName=redirect-to-https@docker middlewareType=RedirectScheme routerName=redirs@docker
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Setting up redirection to https " middlewareName=redirect-to-https@docker middlewareType=RedirectScheme routerName=redirs@docker entryPointName=http
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding tracing to middleware" entryPointName=http routerName=redirs@docker middlewareName=redirect-to-https@docker        traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating middleware" entryPointName=http middlewareName=traefik-internal-recovery middlewareType=Recovery
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="No entryPoint defined for this router, using the default one(s) instead: [http https]" routerName=ping@docker
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Added outgoing tracing middleware api@internal" middlewareName=tracing middlewareType=TracingForwarder entryPointName=https routerName=traefik@docker
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating middleware" entryPointName=https routerName=traefik@docker middlewareName=simpleauth@docker middlewareType=BasicAuth
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding tracing to middleware" entryPointName=https routerName=traefik@docker middlewareName=simpleauth@docker
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating middleware" entryPointName=https routerName=traefik@docker middlewareName=traefik_fixprefix@docker middlewareType=StripPrefix
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding tracing to middleware" middlewareName=traefik_fixprefix@docker entryPointName=https routerName=traefik@docker      traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating middleware" entryPointName=https routerName=whoami@docker serviceName=whoami-whoami middlewareName=pipelining middlewareType=Pipelining
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating load-balancer" serviceName=whoami-whoami entryPointName=https routerName=whoami@docker
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating server 0 http://172.28.0.5:80" entryPointName=https routerName=whoami@docker serviceName=whoami-whoami serverName=0
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Added outgoing tracing middleware whoami-whoami" routerName=whoami@docker entryPointName=https middlewareName=tracing middlewareType=TracingForwarder
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating middleware" middlewareType=StripPrefix entryPointName=https routerName=whoami@docker middlewareName=whoami_fixprefix@docker
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Adding tracing to middleware" entryPointName=https routerName=whoami@docker middlewareName=whoami_fixprefix@docker        traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Added outgoing tracing middleware ping@internal" middlewareName=tracing middlewareType=TracingForwarder entryPointName=https routerName=ping@docker
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating middleware" entryPointName=https middlewareType=Recovery middlewareName=traefik-internal-recovery
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Creating middleware" entryPointName=http middlewareName=traefik-internal-recovery middlewareType=Recovery
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="No entryPoint defined for this router, using the default one(s) instead: [http https]" routerName=ping@docker
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Try to challenge certificate for domain [mydomain.dev] found in HostSNI rule" providerName=leresolver.acme routerName=traefik@docker rule="Host(`mydomain.dev`)&&PathPrefix(`/traefik`)"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Try to challenge certificate for domain [mydomain.dev] found in HostSNI rule" routerName=whoami@docker rule="Host(`mydomain.dev`)&&PathPrefix(`/who`)" providerName=leresolver.acme
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Try to challenge certificate for domain [mydomain.dev] found in HostSNI rule" routerName=ping@docker rule="Host(`mydomain.dev`)&&Path(`/ping`)" providerName=leresolver.acme
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Looking for provided certificate(s) to validate [\"mydomain.dev\"]..." providerName=leresolver.acme routerName=ping@docker rule="Host(`mydomain.dev`)&&Path(`/ping`)"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="No ACME certificate generation required for domains [\"mydomain.dev\"]." providerName=leresolver.acme routerName=ping@docker rule="Host(`mydomain.dev`)&&Path(`/ping`)"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Looking for provided certificate(s) to validate [\"mydomain.dev\"]..." providerName=leresolver.acme routerName=traefik@docker rule="Host(`mydomain.dev`)&&PathPrefix(`/traefik`)"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="No ACME certificate generation required for domains [\"mydomain.dev\"]." routerName=traefik@docker rule="Host(`mydomain.dev`)&&PathPrefix(`/traefik`)" providerName=leresolver.acme
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="Looking for provided certificate(s) to validate [\"mydomain.dev\"]..." providerName=leresolver.acme routerName=whoami@docker rule="Host(`mydomain.dev`)&&PathPrefix(`/who`)"
traefik    | time="2020-02-09T17:43:16Z" level=debug msg="No ACME certificate generation required for domains [\"mydomain.dev\"]." providerName=leresolver.acme routerName=whoami@docker rule="Host(`mydomain.dev`)&&PathPrefix(`/who`)"
// init configuration done
// lets get mydomain.dev/ping
traefik    | time="2020-02-09T17:43:52Z" level=error msg="Recovered from panic in HTTP handler [189.162.136.12:50994 - /ping]: runtime error: invalid memory address or nil pointer dereference" middlewareName=traefik-internal-recovery middlewareType=Recovery
traefik    | time="2020-02-09T17:43:52Z" level=error msg="Stack: goroutine 94 [running]:\ngithub.com/containous/traefik/v2/pkg/middlewares/recovery.recoverFunc(0x29c82c0, 0xc000179aa0, 0x29be840, 0xc000424760, 0xc00021ea00)\n\t/go/src/github.com/containous/traefik/pkg/middlewares/recovery/recovery.go:47 +0x1db\npanic(0x211bd20, 0x3fd7e30)\n\t/usr/local/go/src/runtime/panic.go:679 +0x1b2\ngithub.com/containous/traefik/v2/pkg/ping.(*Handler).ServeHTTP(0x0, 0x29be840, 0xc000424760, 0xc00021ec00)\n\t/go/src/github.com/containous/traefik/pkg/ping/ping.go:31 +0x26\ngithub.com/containous/traefik/v2/pkg/middlewares/tracing.(*forwarderMiddleware).ServeHTTP(0xc000272c60, 0x29be840, 0xc000424760, 0xc00021ec00)\n\t/go/src/github.com/containous/traefik/pkg/middlewares/tracing/forwarder.go:38 +0x610\ngithub.com/containous/traefik/v2/pkg/middlewares/accesslog.(*FieldHandler).ServeHTTP(0xc0005191c0, 0x29be840, 0xc000424760, 0xc00021ec00)\n\t/go/src/github.com/containous/traefik/pkg/middlewares/accesslog/field_middleware.go:29 +0x18d\ngithub.com/gorilla/mux.(*Router).ServeHTTP(0xc00065f800, 0x29be840, 0xc000424760, 0xc00021ec00)\n\t/go/pkg/mod/github.com/containous/mux@v0.0.0-20181024131434-c33f32e26898/mux.go:133 +0x104\ngithub.com/containous/traefik/v2/pkg/middlewares/recovery.(*recovery).ServeHTTP(0xc0002b4be0, 0x29be840, 0xc000424760, 0xc00021ea00)\n\t/go/src/github.com/containous/traefik/pkg/middlewares/recovery/recovery.go:33 +0x131\ngithub.com/containous/traefik/v2/pkg/middlewares/accesslog.(*FieldHandler).ServeHTTP(0xc000519380, 0x29be840, 0xc000424760, 0xc00021ea00)\n\t/go/src/github.com/containous/traefik/pkg/middlewares/accesslog/field_middleware.go:29 +0x18d\ngithub.com/containous/traefik/v2/pkg/middlewares/requestdecorator.(*RequestDecorator).ServeHTTP(0xc00000f8f0, 0x29be840, 0xc000424760, 0xc00021e900, 0xc0003fec48)\n\t/go/src/github.com/containous/traefik/pkg/middlewares/requestdecorator/request_decorator.go:47 +0x37d\ngithub.com/containous/traefik/v2/pkg/middlewares/requestdecorator.WrapHandler.func1.1(0x29be840, 0xc000424760, 0xc00021e900)\n\t/go/src/github.com/containous/traefik/pkg/middlewares/requestdecorator/request_decorator.go:84 +0x83\nnet/http.HandlerFunc.ServeHTTP(0xc0002b51c0, 0x29be840, 0xc000424760, 0xc00021e900)\n\t/usr/local/go/src/net/http/server.go:2007 +0x44\ngithub.com/containous/traefik/v2/pkg/middlewares.(*HTTPHandlerSwitcher).ServeHTTP(0xc00000f8e8, 0x29be840, 0xc000424760, 0xc00021e900)\n\t/go/src/github.com/containous/traefik/pkg/middlewares/handler_switcher.go:23 +0x70\ngithub.com/containous/traefik/v2/pkg/middlewares/forwardedheaders.(*XForwarded).ServeHTTP(0xc000386f50, 0x29be840, 0xc000424760, 0xc00021e900)\n\t/go/src/github.com/containous/traefik/pkg/middlewares/forwardedheaders/forwarded_header.go:174 +0x10b\nnet/http.serverHandler.ServeHTTP(0xc000126460, 0x29be840, 0xc000424760, 0xc00021e900)\n\t/usr/local/go/src/net/http/server.go:2802 +0xa4\nnet/http.initNPNRequest.ServeHTTP(0x29c82c0, 0xc000588900, 0xc000329180, 0xc000126460, 0x29be840, 0xc000424760, 0xc00021e900)\n\t/usr/local/go/src/net/http/server.go:3366 +0x8d\nnet/http.(*http2serverConn).runHandler(0xc0004c3680, 0xc000424760, 0xc00021e900, 0xc0004ee120)\n\t/usr/local/go/src/net/http/h2_bundle.go:5713 +0x9f\ncreated by net/http.(*http2serverConn).processHeaders\n\t/usr/local/go/src/net/http/h2_bundle.go:5447 +0x4eb\n" middlewareName=traefik-internal-recovery middlewareType=Recovery
```
</details>

```
Stack: goroutine 94 [running]:
github.com/containous/traefik/v2/pkg/middlewares/recovery.recoverFunc(0x29c82c0, 0xc000179aa0, 0x29be840, 0xc000424760, 0xc00021ea00)
	/go/src/github.com/containous/traefik/pkg/middlewares/recovery/recovery.go:47 +0x1db
panic(0x211bd20, 0x3fd7e30)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/containous/traefik/v2/pkg/ping.(*Handler).ServeHTTP(0x0, 0x29be840, 0xc000424760, 0xc00021ec00)
	/go/src/github.com/containous/traefik/pkg/ping/ping.go:31 +0x26
github.com/containous/traefik/v2/pkg/middlewares/tracing.(*forwarderMiddleware).ServeHTTP(0xc000272c60, 0x29be840, 0xc000424760, 0xc00021ec00)
	/go/src/github.com/containous/traefik/pkg/middlewares/tracing/forwarder.go:38 +0x610
github.com/containous/traefik/v2/pkg/middlewares/accesslog.(*FieldHandler).ServeHTTP(0xc0005191c0, 0x29be840, 0xc000424760, 0xc00021ec00)
	/go/src/github.com/containous/traefik/pkg/middlewares/accesslog/field_middleware.go:29 +0x18d
github.com/gorilla/mux.(*Router).ServeHTTP(0xc00065f800, 0x29be840, 0xc000424760, 0xc00021ec00)
	/go/pkg/mod/github.com/containous/mux@v0.0.0-20181024131434-c33f32e26898/mux.go:133 +0x104
github.com/containous/traefik/v2/pkg/middlewares/recovery.(*recovery).ServeHTTP(0xc0002b4be0, 0x29be840, 0xc000424760, 0xc00021ea00)
	/go/src/github.com/containous/traefik/pkg/middlewares/recovery/recovery.go:33 +0x131
github.com/containous/traefik/v2/pkg/middlewares/accesslog.(*FieldHandler).ServeHTTP(0xc000519380, 0x29be840, 0xc000424760, 0xc00021ea00)
	/go/src/github.com/containous/traefik/pkg/middlewares/accesslog/field_middleware.go:29 +0x18d
github.com/containous/traefik/v2/pkg/middlewares/requestdecorator.(*RequestDecorator).ServeHTTP(0xc00000f8f0, 0x29be840, 0xc000424760, 0xc00021e900, 0xc0003fec48)
	/go/src/github.com/containous/traefik/pkg/middlewares/requestdecorator/request_decorator.go:47 +0x37d
github.com/containous/traefik/v2/pkg/middlewares/requestdecorator.WrapHandler.func1.1(0x29be840, 0xc000424760, 0xc00021e900)
	/go/src/github.com/containous/traefik/pkg/middlewares/requestdecorator/request_decorator.go:84 +0x83
net/http.HandlerFunc.ServeHTTP(0xc0002b51c0, 0x29be840, 0xc000424760, 0xc00021e900)
	/usr/local/go/src/net/http/server.go:2007 +0x44
github.com/containous/traefik/v2/pkg/middlewares.(*HTTPHandlerSwitcher).ServeHTTP(0xc00000f8e8, 0x29be840, 0xc000424760, 0xc00021e900)
	/go/src/github.com/containous/traefik/pkg/middlewares/handler_switcher.go:23 +0x70
github.com/containous/traefik/v2/pkg/middlewares/forwardedheaders.(*XForwarded).ServeHTTP(0xc000386f50, 0x29be840, 0xc000424760, 0xc00021e900)
	/go/src/github.com/containous/traefik/pkg/middlewares/forwardedheaders/forwarded_header.go:174 +0x10b
net/http.serverHandler.ServeHTTP(0xc000126460, 0x29be840, 0xc000424760, 0xc00021e900)
	/usr/local/go/src/net/http/server.go:2802 +0xa4
net/http.initNPNRequest.ServeHTTP(0x29c82c0, 0xc000588900, 0xc000329180, 0xc000126460, 0x29be840, 0xc000424760, 0xc00021e900)
	/usr/local/go/src/net/http/server.go:3366 +0x8d
net/http.(*http2serverConn).runHandler(0xc0004c3680, 0xc000424760, 0xc00021e900, 0xc0004ee120)
	/usr/local/go/src/net/http/h2_bundle.go:5713 +0x9f
created by net/http.(*http2serverConn).processHeaders
	/usr/local/go/src/net/http/h2_bundle.go:5447 +0x4eb
```

**Comment by ldez** [comment]
Hello,

to use ping, you have to enable it the static configuration

https://docs.traefik.io/v2.1/operations/ping/

**Comment by scratchmex** [comment]
I think this shouldn't be the expected behavior but you are right. This solve the issue. I close

**Comment by traefiker** [comment]
Closed by #6304.
